gen/dcos-config.yaml

```diff
@@ -1125,3 +1117,100 @@ package:
     content: {{ fault_domain_detect_contents }}
 {% case "false" %}
 {% endswitch %}
+  - path: /etc/telegraf.env
+    content: |
+      TELEGRAF_CONFIG_FILE=/opt/mesosphere/etc/telegraf/telegraf.conf
+      TELEGRAF_CONFIG_DIR=/opt/mesosphere/etc/telegraf/telegraf.d/
```

Comment:
What goes in the config dir vs. in the config file?

Reply:
I'm putting the shared configuration for all cluster nodes in `telegraf.conf`, and supplementary config for agents and masters under `telegraf.d/`. That seems to fit the intention behind these options: 

https://docs.influxdata.com/telegraf/v1.7/administration/configuration/#configuration-file-locations
> On most systems, the default locations are `/etc/telegraf/telegraf.conf` for the main configuration file and `/etc/telegraf/telegraf.d` for the directory of configuration files.